lib/puppet_x/templates/init_ps.ps1.erb

```diff
@@ -391,6 +395,12 @@ function Reset-ProcessPowerShellVariables
     [Void]$ps.AddScript($ourFunctions)
     $ps.Invoke()
 
+    if ($WorkingDirectory -ne '') {
```

Comment:
I always tend to overguard on empty strings - `if ($WorkingDirectory -ne $null -and $WorkingDirectory -ne '')`